etc/docker/dev/rucio.cfg

```diff
@@ -182,5 +182,9 @@ dataset_size = 5000000000
 dataset_dir = ~/public/sonar_medium_dataset/
 scope = user.vzavrtan
 
-[permission]
-policy=atlas
+[policy]
+permission = generic
```

Comment:
I am not sure if certain things do not get tested in this case. e.g. localgroupdisk archival policy

Reply:
maybe. it's independent from the tests on travis though so it should not harm for now.